stage2/01-sys-tweaks/01-run.sh

```diff
@@ -32,12 +32,12 @@ systemctl enable resize2fs_once
 EOF
 fi
 
-on_chroot << \EOF
+on_chroot <<EOF
 for GRP in input spi i2c gpio; do
-	groupadd -f -r "$GRP"
+	groupadd -f -r "\$GRP"
 done
 for GRP in adm dialout cdrom audio users sudo video games plugdev input gpio spi i2c netdev; do
-  adduser pi $GRP
+  adduser $FIRST_USER_NAME \$GRP
```

Comment:
Shouldn't `$FIRST_USER_NAME` (and any other of the strings) be double-quoted in case the string contains special characters?

Reply:
For the username it should be limited to a small set of characters anyway. So its best to just check that at the beginning than get so far an pass an invalid username to adduser. See  #235

Reply:
The wpa essid and password needed quoting as well. I added a second patch to #235